index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="https://use.fontawesome.com/382f947f1c.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,700|Raleway:300,500,700" rel="stylesheet">
```

Comment:
Remove Google Fonts. Go to Font Squirrel and generate font kits, and use the trick afterward to avoid faux italics and bolding with your included fonts.